config/api/routes/lock.php

```diff
@@ -6,35 +6,13 @@
  * Content Lock Routes
  */
 return [
```

Comment:
Let's say you want to build a mobile app that still supports content locking. This endpoint would still be needed, right?